api/v1alpha3/machinedeployment_webhook.go

```diff
@@ -78,3 +86,54 @@ func (m *MachineDeployment) validate() error {
 
 	return apierrors.NewInvalid(GroupVersion.WithKind("MachineDeployment").GroupKind(), m.Name, allErrs)
 }
+
+// PopulateDefaultsMachineDeployment fills in default field values
+// This is also called during MachineDeployment sync.
```

Comment:
I moved this function from `defaults.go` to this file since it made sense to have it here instead of a separate standalone file.
I also updated the comment on this function to inform that it is being used elsewhere.